src/kudu/consensus/leader_election.h

```diff
@@ -126,8 +126,8 @@ class FlexibleVoteCounter : public VoteCounter {
   // Vote count per region.
   std::map<std::string, int> yes_vote_count_, no_vote_count_;
 
-  // Leader UUID of the last known leader.
-  const std::string last_known_leader_uuid_;
+  // Last known leader.
+  const LastKnownLeader last_known_leader_;
```

Comment:
I am assuming the usage of the term in LastKnownLeader in the future diff?

Reply:
Yes. This is for the optimization where we have dynamic quorums in single region.